examples/container-service-extension/3.7-cse-step1.tf

```diff
@@ -0,0 +1,274 @@
+# If you have already an Organization, you can remove the `vcd_org` resource from this HCL file and simply
+# configure it in the provider settings.
```

Comment:
It will be not enough as next resource already links to org resource, like: `org  = vcd_org.cse_org.name`. So we should also ask to remove `org  = vcd_org.cse_org.name` everywhere or provide example also with data source were fetch org.

Reply:
Also, if the user already has ORG, is any of the configurations are mandatory to be in some specific state?

Reply:
As per their documentation, no. Just the VDC, that needs to use NSX-T. But I'll re-check just to be 100% sure.

Reply:
Done